.gitignore

```diff
@@ -31,6 +31,7 @@ buildlog
 /omnisharp*.tar.gz
 scripts/Omnisharp*
 .msbuild-*/
+packages/
```

Comment:
ugh, I somehow had that folder in the root. after git clean and rebuild, it's gone, must have been a leftover from a long ago; I will revert this